diff --git a/node_modules/react-native-screens/src/components/ScreenStackItem.tsx b/node_modules/react-native-screens/src/components/ScreenStackItem.tsx
index 43db4ae..b344237 100644
--- a/node_modules/react-native-screens/src/components/ScreenStackItem.tsx
+++ b/node_modules/react-native-screens/src/components/ScreenStackItem.tsx
@@ -104,7 +104,8 @@ function ScreenStackItem(
   let internalScreenStyle;
 
   if (
-    stackPresentationWithDefault === 'formSheet' &&
+    (stackPresentationWithDefault === 'formSheet' ||
+      stackPresentationWithDefault === 'push') &&
     Platform.OS === 'ios' &&
     contentStyle
   ) {
